qa/integration-tests/src/test/java/io/camunda/zeebe/it/multitenancy/MultiTenancyOverIdentityIT.java

```diff
@@ -107,27 +109,14 @@ public class MultiTenancyOverIdentityIT {
           .withNetworkAliases(DATABASE_HOST);
 
   @Container
-  @SuppressWarnings("resource")
-  private static final GenericContainer<?> KEYCLOAK =
-      new GenericContainer<>("bitnami/keycloak:22.0.1")
-          .dependsOn(POSTGRES)
-          .withEnv("KC_HEALTH_ENABLED", "true")
+  private static final KeycloakContainer KEYCLOAK =
+      DefaultTestContainers.createDefaultKeycloak()
           .withEnv("KEYCLOAK_ADMIN_USER", KEYCLOAK_USER)
           .withEnv("KEYCLOAK_ADMIN_PASSWORD", KEYCLOAK_PASSWORD)
-          .withEnv("KEYCLOAK_DATABASE_HOST", DATABASE_HOST)
-          .withEnv("KEYCLOAK_DATABASE_PORT", String.valueOf(DATABASE_PORT))
-          .withEnv("KEYCLOAK_DATABASE_USER", DATABASE_USER)
-          .withEnv("KEYCLOAK_DATABASE_PASSWORD", DATABASE_PASSWORD)
-          .withEnv("KEYCLOAK_DATABASE_NAME", DATABASE_NAME)
+          .withEnv("KEYCLOAK_DATABASE_VENDOR", "dev-mem")
```

Comment:
:thought_balloon: I honestly wasn't sure why they need Postgresql :shrug: That said, I would hope Identity could also work with H2 or some other in memory DB :(